src/repository/resources/asres.rs

```diff
@@ -697,7 +704,7 @@ impl AsBlock {
                 AsRange::skip_content(content)
             }
             else {
-                xerr!(Err(decode::Error::Malformed.into()))
+                Err(content.content_err("invalid AS resources"))
```

Comment:
Do the various "invalid AS resources" messages here and above need to be distinguishable from each other or will it already be apparent which of these code locations created this message?

Reply:
I would want to leave it all as it is for now. We can always (non-breakingly) change this later if it becomes apparent that a simple ‘this is kapot’ won’t do.